docs/kyma-environment-broker/08-05-orchestrate-kyma-upgrade.md

```diff
@@ -13,42 +13,76 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
 
 ## Steps
 
-1. Export these values as environment variables:
-
-   ```bash
-   export RUNTIME_ID={RUNTIME_ID}
-   ```
-
-2. [Get the access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
+1. [Get the access token](#details-authorization). Export this variable based on the token you got from the OAuth client:
 
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
 
-3. Make a call to the Kyma Environment Broker to orchestrate the upgrade.
+2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select runtimes to upgrade using a following selectors:
 
->**NOTE:** The **dry** parameter specified in the request body is set to `true`. As a result, the upgrade is executed in a fake mode. It means that all actions are not executed against selected Runtimes but the Orchestration status is still available.
+- target - can be used to select all runtimes by specifying it as `target: "all"`
+- globalAccount
+- subAccount
+- runtimeID
+- planName
+- region
+
+>**NOTE:** If the **dryRun** parameter specified in the request body is set to `true`, the upgrade is executed but the process is not sending the upgrade request to the Provisioner.
 
    ```bash
-   curl --request POST "https://$BROKER_URL/upgrade/kyma" \
-   --header "$AUTHORIZATION_HEADER" \
-   --header 'Content-Type: application/json' \
-   --data-raw "{
-       \"targets\": {
-           \"include\": {\
-               \"runtime_id\": "uuid-sdasd-sda23t-efs",
-            },
-       },
-        \"dry\": false,
-   }"
+curl --request POST "https://$BROKER_URL/upgrade/kyma" \
+--header "$AUTHORIZATION_HEADER" \
+--header 'Content-Type: application/json' \
+--data-raw "{\
+    \"targets\": {\
+        \"include\": {\
+            \"runtimeID\": \"uuid-sdasd-sda23t-efs\",\
+            \"globalAccount\": \"uuid-sdasd-sda23t-efs\",\
+            \"subAccount\": \"uuid-sdasd-sda23t-efs\",\
+            \"planName\": \"azure\",\
+            \"region\": \"europewest\",\
+         },\
+    },\
+    \"dryRun\": false\
+}"
    ```
 
 A successful call returns the orchestration ID:
 
    ```json
    {
-       "orchestration_id":"8a7bfd9b-f2f5-43d1-bb67-177d2434053c"
+       "orchestrationID":"8a7bfd9b-f2f5-43d1-bb67-177d2434053c"
    }
    ```
 
-4. [Check the operation status](#tutorials-check-orchestration-status).
+4. [Check the orchestration status](#tutorials-check-orchestration-status).
+
+>**NOTE:** Only one orchestration can be processed at the same time. If the KEB is already processing some orchestration the newly created will wait for processing with the **pending** state.
+
+### Strategies
```

Comment:
Shouldn't this section be a part of the general document about orchestration? This is a tutorial and it's not a part of the steps to perform. Let's discuss.